From a6f85e5c90badef94bbcab9a634ea2300f2dbb17 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 10 Jan 2024 11:12:15 +0100
Subject: [PATCH 0833/1141] media: dcmipp: statcap: correct META format naming

Use the new META format naming for ISP statistics

Change-Id: Iad6377233870b53017a9efd4e2f2eff49c24c224
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/353227
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
Reviewed-by: Hugues FRUCHET <hugues.fruchet@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
---
 drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-statcap.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-statcap.c
+++ b/drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-statcap.c
@@ -140,7 +140,7 @@ static int dcmipp_statcap_g_fmt_meta_cap
 {
 	struct v4l2_meta_format *meta = &f->fmt.meta;
 
-	meta->dataformat = V4L2_META_FMT_ST_ISP_STAT;
+	meta->dataformat = V4L2_META_FMT_ST_DCMIPP_ISP_STAT;
 	meta->buffersize = sizeof(struct stm32_dcmipp_stat_buf);
 
 	return 0;
@@ -153,7 +153,7 @@ static int dcmipp_statcap_enum_fmt_meta_
 		return -EINVAL;
 
 	f->type = V4L2_BUF_TYPE_META_CAPTURE;
-	f->pixelformat = V4L2_META_FMT_ST_ISP_STAT;
+	f->pixelformat = V4L2_META_FMT_ST_DCMIPP_ISP_STAT;
 
 	return 0;
 }
